W-16165388 Update policies-included-rate-limiting.adoc (#617)
mulesoft/docs-gateway: gateway/1.8/modules/ROOT/pages/policies-included-rate-limiting.adoc

[NOTE]
====

> A Rate Limiting policy applied to a Flex Gateway API is scoped to replicas, not the gateway. Enable distributed rate limiting to share the request quota across Flex Replicas.

Reviewer: ```suggestion
A Rate Limiting policy applied to a Flex Gateway API is scoped to the replicas, not the gateway. Enable distributed rate limiting to share the request quota across Flex Replicas.
```

| `clusterizable`
| Optional
| `true`

> | Distributed: When using interconnected runtimes with this flag enabled, quota will be shared among all nodes. Distributed rate limiting requires shared storage to be enabled. For more information about shared storage, refer to xref:flex-local-shared-storage-config.adoc[].

Reviewer: ```suggestion
| Distributed: When using interconnected runtimes with this flag enabled, the quota is shared among all nodes. Distributed rate limiting requires shared storage to be enabled. For more information about shared storage, refer to xref:flex-local-shared-storage-config.adoc[].
```

| `fineTuning.distributionMode`
| Optional
| `remaining`

> | Mode of request quota distribution. Sets the distribution algorithm to either service a percentage of the total quota or a percentage of the remaining quota. Supported values are `remaining` `total`.

Reviewer: ```suggestion
| Mode of request quota distribution. Sets the distribution algorithm to either service a percentage of the total quota or a percentage of the remaining quota. Supported values are `remaining` and `total`.
```

|Number of Reqs | The quota available per window | A positive number
|Time Period | The amount of time for which the quota is to be applied | A positive number
|Time Unit | The time in milliseconds, seconds, minutes, or hours | Minutes

> |Distributed | When using multiple Flex Replicas with this flag enabled, the request quota is shared among all nodes. Distributed rate limiting requires shared storage to be enabled. For more information about shared storage, refer to xref:flex-conn-shared-storage-config.adoc[]. | checked or unchecked

Reviewer: ```suggestion
|Distributed | When using multiple Flex Replicas with this flag enabled, the request quota is shared among all nodes. Distributed rate limiting requires shared storage to be enabled. For more information about shared storage, refer to xref:flex-conn-shared-storage-config.adoc[]. | Checked or unchecked
```
